src/strategies/DSL/conditions/priceCondition.ts

```diff
@@ -1,11 +1,51 @@
-import { PrimitiveArgs } from '@brinkninja/types'
-import { PriceConditionArgs } from '@brink-sdk'
+import { PrimitiveArgs, TokenArgs } from '@brinkninja/types'
+import { PriceConditionArgs, UniV3Twap, Token } from '@brink-sdk'
 
-function priceCondition (def: PriceConditionArgs): PrimitiveArgs[] {
+const TIME_INTERVAL = BigInt(1000)
+const OPERATOR_LT = 'lt';
+const OPERATOR_GT = 'gt';
 
-  // TODO: implement
+function priceCondition ({
+  operator,
+  tokenA,
+  tokenB,
+  price
+}: PriceConditionArgs): PrimitiveArgs[] {
+  const twap = new UniV3Twap({
+    tokenA: new Token(toTokenArgs(tokenA)),
+    tokenB: new Token(toTokenArgs(tokenB)),
+    interval: TIME_INTERVAL,
```

Comment:
maybe we can have a DEFAULT_TIME_INTERVAL, which can be 1000, but add an optional param that the user can provide called `twapInterval`

Reply:
also makes sense to add a way for user to optionally set the "fee" (which is not a fee actually, it's the fee tier of the uni v3 pool to use). That's this param https://github.com/brinktrade/brink-sdk/blob/41c34a0b1bae8d5ad08314a7437de78fdda2af8b/src/oracles/UniV3Twap.ts#L28

We could call this `twapFeePool`. The UniV3Twap class already defaults that to `MEDIUM` which makes sense as the default